thoth/metrics_exporter/jobs/pypi.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# thoth-metrics
+# Copyright(C) 2020 Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""PyPI metrics."""
+
+import logging
+
+import requests
+from bs4 import BeautifulSoup
+
+import thoth.metrics_exporter.metrics as metrics
+
+from .base import register_metric_job
+from .base import MetricsBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PyPIMetrics(MetricsBase):
+    """Class to collect statistics from PyPI."""
+
+    @classmethod
+    @register_metric_job
+    def get_pypi_statistics(cls) -> None:
+        """Get statistics from PyPI."""
+        response = requests.get("https://pypi.org/")
```

Comment:
do we need some exception handling here?

Reply:
we run each metric as
```
    try:
        job()
    except Exception:
        _LOGGER.exception("Failed to run metrics gathering job %s.%s", class_name, method_name)
```

Reply:
To have correct reports:

```python
response.raise_for_status()
```